test/core.test.js

```diff
@@ -62,11 +71,11 @@ describe('Core', function () {
 			// Context
 			assert.deepEqual(Object.keys(sent_data.context), ["id","root_id","url"]);
 			assert.ok(guid_re.test(sent_data.context.id), "Request ID is invalid. " + sent_data.context.id);
-			assert.equal(sent_data.context.root_id, "root_id");
+			assert.equal(sent_data.context.root_id, root_id);
 			assert.equal(sent_data.context.url, "http://www.ft.com/home/uk");
 
 			// User
-			assert.deepEqual(Object.keys(sent_data.user), ["user_id"]);
+			assert.deepEqual(Object.keys(sent_data.user), ["passport_id","ft_session","user_id"]);
```

Comment:
I think this was a bug with the old core track implementation.  The documentation says "o-tracking will [...] Automatically pickup ftsession from cookies for you", and I don't think there's an expectation that setting a single user key will unset the defaults, unlike the other arrays.